src/LanguageServer/Protocol/Features/AbstractAnalyzerAssemblyLoaderProvider.cs

```diff
@@ -15,20 +19,53 @@ namespace Microsoft.CodeAnalysis.Host;
 internal abstract class AbstractAnalyzerAssemblyLoaderProvider : IAnalyzerAssemblyLoaderProvider
 {
     private readonly Lazy<IAnalyzerAssemblyLoader> _shadowCopyLoader;
+    private readonly ImmutableArray<IAnalyzerAssemblyResolver> _externalResolvers;
 
     public AbstractAnalyzerAssemblyLoaderProvider(ImmutableArray<IAnalyzerAssemblyResolver> externalResolvers)
     {
         // We use a lazy here in case creating the loader requires MEF imports in the derived constructor.
-        _shadowCopyLoader = new Lazy<IAnalyzerAssemblyLoader>(() => CreateShadowCopyLoader(externalResolvers));
+        _shadowCopyLoader = new Lazy<IAnalyzerAssemblyLoader>(() => CreateShadowCopyLoader(
+#if NET
+            loadContext: null,
+#endif
```

Comment:
i prefer writing things this way on the IDE side vs duplicating the full call, and passing in this parameter on one side.  it keeps the amount of duplication to an absolute minimum.